### DISCUSSION TITLE
Is there a combo box mode?

### DISCUSSION THREAD

**Discussion by sylvain-hamel** [JedWatson/react-select#4712] [Q&A]
Can React Select act as a "combo box" where the user can either select from the list or type text freely (and edit it)? I tried "creatable" but it does not allow the user to fix a typo, they have to create a new value.

**Comment by gormonn** [comment]
"The main feature of this library is providing consumers with the building blocks necessary to create their component."
You need just create component and own state
https://react-select.com/components
